src/main/java/uk/gov/gchq/gaffer/doc/function/DeserialiseXmlExample.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.doc.function;
+
+import uk.gov.gchq.koryphe.impl.function.DeserialiseXml;
+
+public class DeserialiseXmlExample extends FunctionExample {
+    public static void main(final String[] args) {
+        new DeserialiseXmlExample().runAndPrint();
+    }
+
+    public DeserialiseXmlExample() {
+        super(uk.gov.gchq.koryphe.impl.function.DeserialiseXml.class);
+    }
+
+    @Override
+    protected void runExamples() {
+        parseXML();
+    }
+
+    private void parseXML() {
+        // ---------------------------------------------------------
+        final DeserialiseXml function = new DeserialiseXml();
+        // ---------------------------------------------------------
+
+        final String input = "<root><element1><element2 attr=\"attr1\">value1</element2></element1><element1><element2>value2</element2></element1></root>";
+
+        runExample(function, null, input);
```

Comment:
For this one, I'd just add a few more inputs. Maybe try some simpler ones as this can be quite difficult to read, for example:

Input Type | Input | Result Type | Result
-- | -- | -- | --
java.lang.String | &lt;root&gt;&lt;element1&gt;&lt;element2 attr="attr1"&gt;value1&lt;/element2&gt;&lt;/element1&gt;&lt;element1&gt;&lt;element2&gt;value2&lt;/element2&gt;&lt;/element1&gt;&lt;/root&gt; | java.util.HashMap | {root={element1=[{element2={attr=attr1, content=value1}}, {element2=value2}]}}